src/index.html

```diff
@@ -4,9 +4,372 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap"
+      rel="stylesheet"
+    >
+    <link rel="stylesheet" href="main.css">
   </head>
   <body>
-    <h1>Catalog</h1>
+
+    <header>
+      <a href="#home" class="logo"><img src="images/logo.png" alt="logo"></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="https://www.w3schools.com/html/html_formatting.asp"
+              class="nav__link"
+              data-qa="nav-hover">
```

Comment:
Closed bracket should be on the same line as open
```suggestion
            <a
              href="https://www.w3schools.com/html/html_formatting.asp"
              class="nav__link"
              data-qa="nav-hover"
            >
```

Reply:
didn't fix it